src/components/FileTreeNode/styles.module.scss

```diff
@@ -54,6 +54,11 @@
   justify-content: flex-end;
 }
 
+.folderAndFileIcons {
+  flex-shrink: 0;
+  width: 1.1em;
```

Comment:
Since it's not totally obvious why this has a width, can you add a comment saying that this intends to be wider than the widest icon to avoid UI jumpiness? Otherwise, someone may think to remove it.

Reply:
Thanks. I updated it.
I change the width from 1.1em to 1.125em since the expanded folder icon is 1.125em.